### PR TITLE
ztp: add PtpConfig for WPC Dual Cards

### PR DESCRIPTION
**Pull request by jzding**

### CODE REVIEW

File: ztp/source-crs/PtpConfigDualCardsWpc.yaml [hunk 1]
```diff
@@ -0,0 +1,227 @@
+# The grandmaster profile is provided for testing only
```
Comment (aneeshkp) [inline ztp/source-crs/PtpConfigDualCardsWpc.yaml:1]: rename file to s  PtpConfigDualCardGmWpc.yaml

File: ztp/source-crs/PtpConfigDualCardsWpc.yaml [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    ts2phcConf: |
+      [nmea]
+      ts2phc.master 1
+      [global]
+      use_syslog  0
+      verbose 1
+      logging_level 7
+      ts2phc.pulsewidth 100000000
+      #GNSS module s /dev/ttyGNSS* -al use _0
+      #cat /dev/ttyGNSS_1700_0 to find available serial port
```
Comment (aneeshkp) [inline ztp/source-crs/PtpConfigDualCardsWpc.yaml:103]: for WPC it is /dev/GNSS  to find available serial port

File: ztp/source-crs/PtpConfigDualCardsWpc.yaml [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+    ts2phcOpts: " "
+    ts2phcConf: |
+      [nmea]
+      ts2phc.master 1
+      [global]
+      use_syslog  0
+      verbose 1
+      logging_level 7
+      ts2phc.pulsewidth 100000000
+      #GNSS module s /dev/ttyGNSS* -al use _0
```
Comment (aneeshkp) [inline ztp/source-crs/PtpConfigDualCardsWpc.yaml:102]: remove this comment, not sure we use it

File: ztp/source-crs/PtpConfigDualCardsWpc.yaml [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+      #this is a measured value in nanoseconds to compensate for SMA cable delay
+      ts2phc.extts_correction -25
+    ptp4lConf: |
+      [$iface_master]
+      masterOnly 1
+      [$iface_master_1]
+      masterOnly 1
+      [$iface_master_2]
+      masterOnly 1
+      [$iface_master_3]
```
Comment (aneeshkp) [inline ztp/source-crs/PtpConfigDualCardsWpc.yaml:122]: There should be $iface_master_1_1 and $iface_master_1_2
$iface_master_1 of $iface_master is confused with 2nd interface $iface_master_1

File: ztp/source-crs/PtpConfigDualCardsWpc.yaml (hunk 2 above)
Comment (jzding) [reply ztp/source-crs/PtpConfigDualCardsWpc.yaml]: done

File: ztp/source-crs/PtpConfigDualCardsWpc.yaml (hunk 3 above)
Comment (jzding) [reply ztp/source-crs/PtpConfigDualCardsWpc.yaml]: done

**Review by aneeshkp — APPROVED** [review]
/lgtm

**Review by sakhoury — APPROVED** [review]
/lgtm

**Pull request merged**